src/applications/bmqstoragetool/m_bmqstoragetool_recordprinter.cpp

```diff
@@ -155,7 +155,7 @@ void printRecord(bsl::ostream&                  stream,
         }
     }
 
-    bmqu::AlignedPrinter printer(stream, &fields);
+    bmqu::AlignedPrinter printer(stream, fields);
```

Comment:
Also need to change type above:
`bsl::vector<const char*> fields` -> `bsl::vector<bsl::string> fields`